src/entry_table.rs

```diff
@@ -23,6 +23,9 @@ use crate::{
     KeyingMaterial, Keyword, CHAIN_TABLE_KEY_DERIVATION_INFO,
 };
 
+/// Maximum number of Chain Table UID associated to a given Entry Table UID.
+const MAX_UID_PER_CHAIN: usize = 1_000_000;
```

Comment:
I suppose 1_000_000 is a reasonable limit. What happens if this limit is reached at one point? no way to override it right?

Reply:
I think Findex will be unusable way before this limit (timing / size wise). Users shouldn't index so much locations inside the same keyword.